doc/search-metrics.org

```diff
@@ -118,7 +125,10 @@ GET /ctia/judgement/cardinality?aggregate-on=obsevable.value&from=2020-01-01
 
 It will return an integer that represent the estimated number of unique values
 #+BEGIN_SRC javascript
-5391
+{"data": {"observable": {"value": 5384}},
```

Comment:
same remark about value here. Unless perhaps you would like to add redundancy by
also providing a `type` along `value` here it would be `"type":"integer"` for example.
But I think you could get rid of `"value"` here.